apps/mobile/src/store/settings/settings.write.ts

```diff
@@ -69,6 +71,19 @@ export const settingsSlice = createSlice({
     userChangedLanguagePreferenceSource(state, action: PayloadAction<LanguagePreferenceSource>) {
       state.languagePreferenceSource = action.payload;
     },
+    userChangedAssetVisibility(
+      state,
+      action: PayloadAction<{
+        assetId: SerializedCryptoAssetId;
+        value: boolean;
+      }>
+    ) {
+      const updatedAssetVisibility = produce(state.assetVisibility, draftAssetVisibility => {
```

Comment:
This is a bit tricky to follow. Maybe the `const updatedAssetVisibility = produce(` can be extracted from here? 

All the other settings have a crisper format:
```
    userChangedLanguagePreferenceSource(state, action: PayloadAction<LanguagePreferenceSource>) {
      state.languagePreferenceSource = action.payload;
    },
```